diesel_codegen/src/database_url.rs

```diff
@@ -29,8 +29,8 @@ fn load_dotenv_file() -> Result<(), String> {
 
 #[cfg(not(feature = "dotenv"))]
 fn load_dotenv_file() -> Result<(), String> {
-    Err(String::from("The dotenv feature is required to use strings starting \
-        with `dotenv:`"))
+    Err(String::from("diesel_codegen must be compiled with the dotenv feature \
+        to use strings starting with `dotenv:`"))
```

Comment:
Good intuition to not expose too many internals!

But… Hm. Dotenv reads environment variables (from the actual environment and from a `.env` file). The `dotenv:FOO` syntax means "replace this with the environment variable `FOO`". So I'm not sure what it could be doing if it's not substituting a string 😅

"Interpreting" is good, too, but a bit vaguer. I like that it anthropomorphizes diesel_codegen a bit, though! :)